src/Api/AdminConsole/Controllers/OrganizationUsersController.cs

```diff
@@ -691,6 +699,56 @@ public async Task PatchBulkEnableSecretsManagerAsync(Guid orgId,
         await BulkEnableSecretsManagerAsync(orgId, model);
     }
 
+
+    [Authorize<ManageUsersRequirement>]
+    [HttpPost("{id}/auto-confirm")]
+    public async Task<IActionResult> AutomaticallyConfirmOrganizationUserAsync([FromRoute] Guid orgId,
+        [FromRoute] Guid id,
+        [FromBody] OrganizationUserConfirmRequestModel model)
+    {
+        if (!_featureService.IsEnabled(FeatureFlagKeys.AutomaticConfirmUsers))
```

Comment:
This should be replaced with `[RequireFeature(FeatureFlagKeys.AutomaticConfirmUsers)]`